src/main/java/com/cloudbees/jenkins/Cleaner.java

```diff
@@ -52,10 +56,13 @@ protected void doRun() throws Exception {
         }
 
         // subtract all the live repositories
-        for (AbstractProject<?,?> job : Hudson.getInstance().getAllItems(AbstractProject.class)) {
-            GitHubPushTrigger trigger = job.getTrigger(GitHubPushTrigger.class);
-            if (trigger!=null) {
-                names.removeAll(GitHubRepositoryNameContributor.parseAssociatedNames(job));
+        for (Job<?,?> job : Jenkins.getInstance().getAllItems(Job.class)) {
```

Comment:
I think you'll need to add a `null` check on `Jenkins.getInstance()`. Otherwise, we'll be getting FindBugs errors I think.